scalding-core/src/main/scala/com/twitter/scalding/CascadingMode.scala

```diff
@@ -0,0 +1,208 @@
+package com.twitter.scalding
```

Comment:
Are there changes to these classes or were they just moved?

Reply:
just moved.